js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -155,6 +151,14 @@ var KeyboardNavigationController = core.ViewController.inherit({
         });
     },
 
+    _setRowsViewAttributes: function($rowsView) {
+        let isGridEmpty = !this._dataController.getVisibleRows().length;
```

Comment:
Maybe we should use 'const' for all immutable variables for consistently style

Reply:
Sure, I'll correct it